### PR TITLE
PEP8 & autoformat; import fixes (since you moved to src/ folder);

### PR DESCRIPTION
**Pull request by SamuelMarks**
added missing import

### CODE REVIEW

File: bin/decompile [hunk 1]
```diff
@@ -30,20 +30,20 @@
...
 # Standard lib imports
-from os.path import abspath, dirname, join
-import sys
 import argparse
 import logging
+import sys
+from os.path import abspath, dirname, join
 
 # Prepend ../src to $PATH so the project modules can be imported below
 src_path = join(dirname(abspath(__file__)), "../src")
```
Comment (0xl3x1) [inline bin/decompile]: These lines will need to be removed now that imports are in the form `src.blah`

**Comment by 0xl3x1** [comment]
Thanks for this! I notice the Travis build is failing because of the import path being incorrect in `bin/*.py`. If you remove lines 39 and 40 then I believe it should work. :+1:

**Comment by SamuelMarks** [comment]
Yaaa also the directory structure really needs to be conformed to python standards.

Setuptools/disttools style with proper setup.py and all that. Then maybe move to a `python -m vandal -c disassemble -s <filename>` syntax also

**Comment by 0xl3x1** [comment]
@SamuelMarks Yep! I think it would be best to do that in a separate PR though. Are you intending to work on packaging (so we don't double-up)?

**Comment by SamuelMarks** [comment]
Spent about 5 minutes playing around: https://github.com/SamuelMarks/vandal/tree/setuptools (from my [python-package-gen](https://github.com/SamuelMarks/python-package-gen) scaffold generator)

Yeah, I can probably concentrate on packaging. But note that my changes will affect essentially every file in the repo…

**Comment by 0xl3x1** [comment]
Cool, thanks! That's fine, provided you can merge it into master again afterwards ;)
(that's also the reason I suggested using a separate PR)

**Review by 0xl3x1 — APPROVED** [review]
Looks good to me. Thanks for contributing!

**Pull request merged**